tests/test_on_data/test_sorting_interfaces.py

```diff
@@ -44,6 +44,24 @@ class TestCellExplorerSortingInterface(SortingExtractorInterfaceTestMixin, TestC
         dict(
             file_path=str(DATA_PATH / "cellexplorer" / "dataset_3" / "20170519_864um_900um_merge.spikes.cellinfo.mat")
         ),
+        dict(
+            file_path=str(
+                DATA_PATH
+                / "cellexplorer"
+                / "dataset_4"
+                / "Peter_MS22_180629_110319_concat_stubbed"
+                / "Peter_MS22_180629_110319_concat_stubbed.spikes.cellinfo.mat"
+            )
+        ),
```

Comment:
Similar to comment on other PR; can you break these into separate classes per test file and add the metadata check to showcase/assert what is being extracted?

Reply:
They have the same data so I added a single test.